Assets/Mirror/Runtime/NetworkBehaviour.cs

```diff
@@ -74,6 +74,14 @@ protected void InitSyncObject(SyncObject syncObject)
             syncObjects.Add(syncObject);
         }
 
+        private static NetworkWriter argWriter;
```

Comment:
what if we just make this one protected static, initialize it already and accesses it from Weaver?
this would be the same way we cache NetworkWriters elsewhere.

Reply:
true,  we can do that.

Reply:
I think it would be less magic this way

Reply:
I started doing this change and I realized that I needed to call SetLength(0) from the weaver instead every time.

I don't want to do code in the weaver that can be done in the runtime class.

Reply:
GetArgWriter still looks a bit weird though.
can we just create argWriter in any case instead of in that function?
what about a better name like weaverWriter or weaverRemoteArgWriter or similar?

Reply:
Remember that #911 also deals with this, if you're going to merge that, then don't think about this